modules/adding-node-iso-configs.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/nodes-nodes-adding-node-iso.adoc
+
+[id="adding-node-iso-configs_{context}"]
+= Cluster configuration reference
+
+When creating the ISO image, configurations are retrieved from the target cluster and are applied to the new nodes.
+Any configurations for your cluster are applied to the nodes unless you override the configurations in either the `nodes-config.yaml` file or any flags you add to the `oc adm node-image create` command.
+
+[id="adding-node-iso-yaml-config_{context}"]
+== YAML file parameters
+
+Configuration parameters that can be specified in the `nodes-config.yaml` file are described in the following table:
+
+[cols=".^4l,.^3,.^2",options="header"]
+|====
+|Parameter|Description|Values
+
+|hosts:
+|Host configuration.
+|An array of host configuration objects.
+
+|hosts:
+  hostname:
+|Hostname.
+Overrides the hostname obtained from either the Dynamic Host Configuration Protocol (DHCP) or a reverse DNS lookup.
+Each host must have a unique hostname supplied by one of these methods, although configuring a hostname through this parameter is optional.
+|String.
+
+|hosts:
+  interfaces:
+|Provides a table of the name and MAC address mappings for the interfaces on the host.
+If a `NetworkConfig` section is provided in the `nodes-config.yaml` file, this table must be included and the values must match the mappings provided in the `NetworkConfig` section.
+|An array of host configuration objects.
+
+|hosts:
+  interfaces:
+    name:
+|The name of an interface on the host.
+|String.
+
+|hosts:
+  interfaces:
+    macAddress:
+|The MAC address of an interface on the host.
+|A MAC address such as the following example: `00-B0-D0-63-C2-26`.
+
+|hosts:
+  role:
+|Defines whether the host is a `master` or `worker` node.
+Currently, only worker nodes can be added using this process.
+|`worker`.
```

Comment:
I have no idea whether this sort of parameter would be available, given that only worker nodes are supported for now, but I included it just in case

Reply:
Since for now we do support only workers (and essentially it's ignored), let's remove it from the docs